src/Core/Dirt/Reports/ReportFeatures/AddOrganizationReportCommand.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Bit.Core.Dirt.Entities;
+using Bit.Core.Dirt.Reports.ReportFeatures.Interfaces;
+using Bit.Core.Dirt.Reports.ReportFeatures.Requests;
+using Bit.Core.Dirt.Repositories;
+using Bit.Core.Exceptions;
+using Bit.Core.Repositories;
+
+namespace Bit.Core.Dirt.Reports.ReportFeatures;
```

Comment:
🌱 In the future we might want to break the reports up into their own directories.